AppController/djinn.rb

```diff
@@ -1778,6 +1786,18 @@ def job_start(secret)
           scale_appservers_across_nodes()
         end
       }
+
+      # We want to print some statistic in the logs every hour or so, to
+      # ensure that the thread is working correctly.
+      if last_print < (Time.now.to_i - 60 * PRINT_STATS_MINUTES)
```

Comment:
This constant would be to identify the number of seconds per minute, and will only be used here (there is another 60 in the code, but the context is slightly different). Given that the unit is fixed (60) it seems very verbose and redundant to me to create a constant for it , since everyone knows that unit. Thoughts?